File: CH1-input-process-output/EX9-temperature-converter/tempConvert.cpp
#include <iostream>
using namespace std;

int main()
{
    int celcius;
    int farenheit;

    // Input
    cout << "Enter a temperature(celcius) :";
    cin >> celcius;

    // Process
    farenheit = (celcius * 9 / 5) + 32;

    // Output
    cout << "Temperature is " << farenheit << " farenheit" << endl;

    return 0;
}

File: CH1-input-process-output/EX9-temperature-converter/source-code/tempConvert.js
const prompt = require("prompt-sync")();

var celcius, farenheit;

// input
celcius = Number(prompt("Enter a Temperature(Celcius) :"));

// Process
farenheit = (celcius * 9) / 5 + 32;

// Output
console.log("Temperature is " + farenheit + " farenheit");
